samples/snippets/csharp/VS_Snippets_CLR/conceptual.resources.locating/cs/assemblyinfo.cs

```diff
@@ -1,40 +0,0 @@
-﻿using System.Reflection;
```

Comment:
As *assemblyinfo.cs* files are deleted, should we explicitly set [GenerateAssemblyInfo](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#generateassemblyinfo) to False in csproj?

Reply:
> The documentation says the default value is True, so I'd expect them to be re-generated, is that correct?

Right. These deleted *AssemblyInfo.cs* files only contain auto-generated values.